integration-tests/src/test/resources/docker-compose.yml

```diff
@@ -52,3 +52,10 @@ services:
     command: "mysqld --ssl=0
     --character-set-server=utf8mb4
     --collation-server=utf8mb4_unicode_ci --transaction-isolation=READ-COMMITTED --innodb_autoinc_lock_mode=2"
+  postgres:
+    image: postgres:10.4
```

Comment:
Lets up to 12, we have only couple of database not upgraded to it.

Reply:
👌